base_layer/core/src/base_node/states/fetching_horizon_state.rs

```diff
@@ -111,6 +112,11 @@ impl HorizonInfo {
             return StateEvent::FatalError(format!("Synchronizing UTXO set failed. {}", e));
         }
 
+        info!(target: LOG_TARGET, "Validation synced horizon state.");
+        if let Err(e) = self.validate_horizon_state(shared).await {
+            return StateEvent::FatalError(format!("Validation of synced horizon state failed. {}", e));
+        }
+
         debug!(target: LOG_TARGET, "Pruning horizon state has synchronised");
```

Comment:
Maybe this should be `info` as well to keep it consistent with the type of messages emitted above

Reply:
Change applied.